src/agents/combine_audio_files_agent.ts

```
@@ -94,6 +94,7 @@ const getInputIds = (context: MulmoStudioContext, mediaDurations: MediaDuration[
     const { silenceDuration } = mediaDurations[index];
     const paddingId = `[padding_${index}]`;
     if (studioBeat.audioFile) {
+      assert(isFile(studioBeat.audioFile), `studioBeat.audioFile is not exist or not file: index=${index}`); 
```

ESLint

[error] Delete `·`

(prettier/prettier)